Library/Formula/memo.rb

```diff
@@ -0,0 +1,20 @@
+class Memo < Formula
+  url "http://www.getmemo.org/memo-1.4.tar.gz"
+  homepage "http://www.getmemo.org"
+  sha1 "0b4c1b22fac0644b54d7de1e89cb20a54c8fff25"
+
+
+  def install
+    system "make", "PREFIX=#{prefix}"
+    bin.install "memo"
```

Comment:
if I `make install`, brew asks me overwrite all conflicting files, I overwritten them and it still works.